<launch>

    <!-- Load joint controller configurations from YAML file to parameter server -->
    <rosparam file="$(find pmc6r_control)/config/pmc6r_control.yaml" command="load"/>

    <!-- load the controllers -->
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" ns="/pmc6r" args="joint_state_controller
                                          joint_position_controller
                                          "/>
    <!--
                                          arm_controller
                                          joint1_position_controller
                                          joint2_position_controller
                                          joint3_position_controller
                                          joint4_position_controller
                                          joint5_position_controller
                                          joint6_position_controller
    -->

    <!-- convert joint states to TF transforms for rviz, etc -->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
        respawn="false" output="screen">
        <remap from="/joint_states" to="/pmc6r/joint_states" />
    </node>

    <!-- experiment to control as group, not working now
    <node name="simple_mover" pkg="pmc6r_control" type="simple_mover.py"/>
    -->

</launch>
